mjpc/tasks/allegro/allegro.cc

```diff
@@ -95,7 +96,29 @@ void AllegroCube::TransitionLocked(mjModel* model, mjData* data) {
     mj_forward(model, data);
     mutex_.lock();
   }
+}
+
+// Change the friction coefficient of all the objects in the scene
+void AllegroCube::DomainRandomize(std::vector<mjModel*>& randomized_models) 
```

Comment:
Are there other things we can try to domain randomize besides friction here?

Reply:
Just tried with actuator gains - that seems to help considerably in sim. There are surely plenty of other things we can do too, it's just a matter of figuring out the mujoco syntax

Reply:
That's pretty interesting! I think this would be a great thing to domain randomize over because we already know there's substantial mismatch between that and our model already